### DISCUSSION TITLE
Is there any way we can make first row sticky along with header ?

### DISCUSSION THREAD

**Discussion by yashwantsinghzala** [namespace-ee/react-calendar-timeline#990] [Q&A]
**Is your feature request related to a problem? Please describe.**
we have requirement to make first row sticky along with header which is basically a user row .

**Describe the solution you'd like**
I want to know solution or workaround to make very first row sticky along with header . 

**Describe alternatives you've considered**

**Additional context**

**Comment by apigon** [comment]
Hi, were you able to resolve that requirement any way? I'm facing same issue right now.
